src/main/resources/assets/quark/lang/ru_ru.json

```
@@ -175,137 +175,135 @@
 
     "quark.jei.makes_copy": "Делает копию",
     "quark.jei.any_enchanted": "Любой зачарованный предмет",
-    "quark.jei.influence": "Матрица влияния",
+    "quark.jei.influence": "Матрица зачарований",
```

```suggestion
    "quark.jei.influence": "Матрица чар",
```
На 283 строке, матрица стала "Матрица чар"

Всё остальное нормально